packages/server-wallet/src/wallet/store.ts

```diff
@@ -162,6 +163,10 @@ export class Store {
     });
   }
 
+  async transaction<T>(callback: (tx: Transaction) => T): Promise<T> {
+    return this.knex.transaction(async tx => callback(tx));
+  }
```

Comment:
Is there a chance this is unrelated to this PR, and sneaking in?

Reply:
This is used in the test setup. We'll also need it in other places going forward though.